python/lsst/ap/association/dia_object.py

```diff
@@ -55,12 +70,11 @@ class DIAObject(object):
         of DIASources changes and set to true when the initialize method is
         run.
     """
-
     def __init__(self, dia_source_catalog, object_source_record=None):
```

Comment:
This is for the case where you have already created a DIAObject and don't want to rerun the summary statistics on it's corresponding DIASourceCatalog. This would happen, for instance, when you persist DIAObjects and load them on a subsequent visit.